projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/index.native.js

```diff
@@ -72,7 +72,8 @@ const VideoPressUploader = ( { handleDoneUpload } ) => {
 			onSelect={ onSelectVideo }
 			onSelectURL={ onSelectURL }
 			allowedTypes={ VIDEOPRESS_VIDEO_ALLOWED_MEDIA_TYPES }
-			autoOpenMediaUpload
+			autoOpenMediaUpload={ autoOpenMediaUpload }
+			onFocus={ onFocus }
```

Comment:
We pass `onFocus` to `MediaPlaceholder` to automatically select the block when the placeholder is tapped ([reference](https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-editor/src/components/media-placeholder/index.native.js#L184-L187)).